README.md

```diff
@@ -1,17 +1,38 @@
-# PCSX2 (Flathub)
-net.pcsx2.PCSX2
+# Flatpak for PCSX2 (on Flathub)
+[PCSX2 Official Website](https://pcsx2.net)
 
-[PCSX2 Official Website](https://pcsx2.org)
+## Install
 
-## Building
-```bash
-$> flatpak-builder builddir --force-clean --install-deps-from=flathub net.pcsx2.PCSX2.json
-```
+(Your operating system's "app store" or application installer may provide a graphical way to install, run, and uninstall flatpaks, as an alternative to entering some of the commands below in a terminal window.)
+
+0. [Dump your Playstation 2's BIOS](https://pcsx2.net/guides/basic-setup/).
+
+1. [Set up Flatpak](https://www.flatpak.org/setup/).
+
+2. Install PCSX2 from [Flathub](https://flathub.org/apps/details/net.pcsx2.PCSX2).
+
+   `flatpak install -y net.pcsx2.PCSX2`
+
+3. Start PCSX2.
+
+   `flatpak run net.pcsx2.PCSX2`
+
+4. Copy your PS2 BIOS files to PCSX2's data directory.
+
+   When the "PCSX2 First Time Config" dialog prompts you to "Select a BIOS rom", copy the BIOS files to the default BIOS Search Path it shows, probably `$HOME/.var/app/net.pcsx2.PCSX2/config/PCSX2/bios`. Then  click [Refresh list] and PCSX2 should find your BIOS.
```

Comment:
(What happened to my comment, GitHub?) I was explicit about timing because the directory path doesn't exist until this step. I verified that creating it in advance with `mkdir -p` works but, it's prone to error. Since the "Select a BIOS rom" step is somewhat self-documenting if the user messes up, I changed to your way but left a shorter description of what should happen.